portfolio/src/main/java/com/google/sps/servlets/ServerCommentsServlet.java

```diff
@@ -31,26 +35,48 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get & store the input from the comment section.
     String userName = getParameter(request, "name", "Anonymous");
     String userComment = getParameter(request, "comment", "");
-    comments.add(new CommentDetails(userName, userComment));
-    
-    // Convert the comments to JSON
-    String commentJson = convertToJson(comments);
 
-    // Respond with the result.
-    response.setContentType("application/json;");
-    response.getWriter().println(commentJson);
+    Entity commentEntity = new Entity("CommentDetails");
+    commentEntity.setProperty("name", userName);
+    commentEntity.setProperty("comment", userComment);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+    
+    response.sendRedirect("/contact.html");
   }
 
   // Used to fetch data in frontend 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+    List<CommentDetails> comments = new ArrayList<>();
```

Comment:
No need to initialize `comments` with an ArrayList that is never used. You can just write

`List<...> comments = extractComments();`